src/jetson/main.py

```diff
@@ -297,28 +304,28 @@ def encryptFrame(img:np.ndarray,
     g.eval()
     class_names = ['Glasses', 'Goggles', 'Neither']
 
-    cap = VideoCapturer() #Instantiate Video Capturer object
+    cap = VideoCapturer() 
     detector = FaceDetector(detector=args.detector, cuda=args.cuda and torch.cuda.is_available(), set_default_dev=True) #Instantiate Face Detector object
-    cl = Classifier(g) #Instantiate Classifier object
+    cl = Classifier(g)
+    enc = Encryptor() 
```

Comment:
can we make these variable names more descriptive? `cap` , `c1` and `enc` are hard to read